src/mixins/portfolio.js

```diff
@@ -15,6 +16,10 @@ const tabOptions = {
   created: 'created',
 };
 
+const ITEMS_PER_PAGE = 10;
+
+const fetchNFTInfoLimit = throat(10);
```

Comment:
is this 10 related to ITEMS_PER_PAGE?

Reply:
Not directly, it is the number of concurrent NFT metadata requests